libraries/provision/ansible/playbooks/tasks/setup-env-variables.yml

```diff
@@ -2,14 +2,11 @@
 - name: SYNC GATEWAY | setting up environment variables for service on centos
   become: yes
   file: path=/etc/systemd/system/sync_gateway.service.d state=directory
-  when: ansible_distribution == "CentOS" or ansible_distribution == "RedHat"
 
```

Comment:
by removing the condition, how ansible script know if this is centos or redhat platform and perform these tasks? or the tasks are now for all platforms? or this yml file will only be run on centos/redhat setup?

Reply:
setup-env-variables.yml is called from setup-env-variables-for-service.yml which is already have condition to use for centos, redhat, debian , ubuntu, so this condition does not make sense to have as top level is taking care